modules/combined/tests/1D_axisymmetric/axisymmetric_gps_incremental.i

```diff
@@ -1,48 +1,62 @@
 #
-# This test checks the out of plane stress and constraint.  The model consists
-# of two sets of line elements.  One undergoes a temperature rise of 100 with
+# This test checks the generalized plane strain using incremental small strain formulation.
+# The model consists of two sets of line elements. One undergoes a temperature rise of 100 with
 # the other seeing a temperature rise of 300.  Young's modulus is 3600, and
 # Poisson's ratio is 0.2.  The thermal expansion coefficient is 1e-8.  All
 # nodes are constrained against movement.
 #
-# Without the strain_yy constraint, the stress solution would be
-# [-6e-3, -6e-3, -6e-3] and [-18e-3, -18e-3, -18e-3] (xx, yy, zz).  The out
-# of plane stress kernel works to minimize the out of plane stress, and the
-# constraint forces the out of plane strain to be equal.
+# For plane strain case, i.e., without constraining the strain_yy to be uniform,
+# the stress solution would be [-6e-3, -6e-3, -6e-3] and [-18e-3, -18e-3, -18e-3] (xx, yy, zz).
+# The generalized plane strain kernels work to balance the force in y direction.
 #
-# With an out of plane strain of 3e-3, the stress solution becomes
+# With out of plane strain of 3e-6, the stress solution becomes
 # [-3e-3, 6e-3, -3e-3] and [-15e-3, -6e-3, -15e-3] (xx, yy, zz).  This gives
-# an average out of plane stress of zero.
+# a domain integral of out-of-plane stress to be zero.
 #
 
 [GlobalParams]
-  # Set initial fuel density, other global parameters
-  order = FIRST
-  family = LAGRANGE
   displacements = disp_x
+  scalar_strain_yy = scalar_strain_yy
```

Comment:
I don't think we should hard code a coordinate direction into this parameter name.